validation/pipeline/tfp/Jenkinsfile.harvester.e2e

```diff
@@ -112,7 +112,7 @@ node("harvester-vpn-1") {
                         echo "harvester IP address"
                         echo $inode
 
-                        sleep 240
+                        sleep 230
```

Comment:
What's the reason this decreased by 10 seconds?

Reply:
to save 10 seconds of time 😄 this time is just based on observations